modules/kobld_billboards/root.sqf

```diff
@@ -0,0 +1,9 @@
+#ifdef preinit
```

Comment:
```sqf
#ifdef description_XEH_PreInit
	class Billboards {
		serverInit = "'' call compile preprocessFileLineNumbers 'modules\kobld_billboards\preinitServer.sqf'";
	};
#endif
```

Reply:
Named the class slightly differently but fixed.